Problem building Eclipse SmartHome

Tried to build Eclipse SmartHome with following results:

results from mvn -v

Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T08:41:47-08:00)
Maven home: /home/lpyle/maven/apache-maven-3.3.9
Java version: 1.8.0_72, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-8-oracle/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "3.13.0-77-generic", arch: "amd64", family: "unix"

results from mvn clean install:

.......
[INFO] Resolving dependencies of MavenProject: org.eclipse.smarthome:org.eclipse.smarthome.feature.runtime.automation:0.8.0-SNAPSHOT @ /home/lpyle/smarthome/features/org.eclipse.smarthome.feature.runtime.automation/pom.xml
[INFO] {osgi.os=linux, osgi.ws=gtk, org.eclipse.update.install.features=true, osgi.arch=x86}
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: org.eclipse.smarthome.feature.runtime.automation.feature.group 0.8.0.qualifier
[ERROR]   Missing requirement: org.eclipse.smarthome.feature.runtime.automation.feature.group 0.8.0.qualifier requires 'org.eclipse.smarthome.automation.sample.handler 0.0.0' but it could not be found
[ERROR] 
[ERROR] See http://wiki.eclipse.org/Tycho/Dependency_Resolution_Troubleshooting for help.
[ERROR] Cannot resolve dependencies of MavenProject: org.eclipse.smarthome:org.eclipse.smarthome.feature.runtime.automation:0.8.0-SNAPSHOT @ /home/lpyle/smarthome/features/org.eclipse.smarthome.feature.runtime.automation/pom.xml: See log for details -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException

Any assistance would be appreciated.

--- a/features/org.eclipse.smarthome.feature.runtime.automation/feature.xml
+++ b/features/org.eclipse.smarthome.feature.runtime.automation/feature.xml
@@ -95,13 +95,6 @@ Automation support
          unpack="false"/>
 
    <plugin
-         id="org.eclipse.smarthome.automation.sample.handler"
-         download-size="0"
-         install-size="0"
-         version="0.0.0"
-         unpack="false"/>
-
-   <plugin
          id="org.eclipse.smarthome.automation.module.timer"
          download-size="0"
          install-size="0"

